src/CrawlTestLvl/private/groupobjbox.h

```diff
@@ -0,0 +1,28 @@
+//#
+//# Copyright (C) 2021-2021 QuasarApp.
+//# Distributed under the GPLv3 software license, see the accompanying
+//# Everyone is permitted to copy and distribute verbatim copies
+//# of this license document, but changing it is not allowed.
+//#
+
+#ifndef GROUPOBJBOX_H
+#define GROUPOBJBOX_H
+
+#include "Crawl/controlpos.h"
+#include "Crawl/iworlditem.h"
+
+namespace TestLvl {
+
+class GroupObjBox: public CRAWL::ControlPos, public CRAWL::IWorldItem {
```

Comment:
this is does not work because the GroupObjBox class has multiple implementation of iworlditems fields from the box  and  the clasterItem classes.
For fix this behavior you need to remove this class and change the box class.
in the Box class you must inherit from the ClasterItem instead the iworldItem class.